raiden/raiden_service.py

```diff
@@ -149,16 +152,17 @@ def __init__(self, chain, private_key_bin, transport, discovery, config):
 
         self.channelgraphs = dict()
         self.manager_token = dict()
+        self.swapkeys_tokenswaps = dict()
+        self.swapkeys_greenlettasks = dict()
 
-        self.hashlock_token_task = defaultdict(dict)
         self.identifier_statemanager = defaultdict(list)
         self.identifier_result = defaultdict(list)
 
         # This is a map from a hashlock to a list of channels, the same
-        # hashlock can be used in more than one token (for exchanges), a
+        # hashlock can be used in more than one token (for tokenswaps), a
         # channel should be removed from this list only when the lock is
         # released/withdrawn but not when the secret is registered.
-        self.hashlock_channel = defaultdict(lambda: defaultdict(list))
+        self.tokens_hashlocks_channels = defaultdict(lambda: defaultdict(list))
```

Comment:
We need to have this in `contributing.md` then. I missed the part where these naming conventions for types could be mixed/matched. I will add it later with this variable name as an example.

But to be honest I would not have deduced it's a `dict(dict(list)))` from the name alone. May need to reconsider this one.